tests/Moq.Tests/CallbacksFixture.cs

```diff
@@ -474,6 +474,105 @@ public void CallbackWithRefParameterCannotModifyNonRefParameter()
 			Assert.Equal("input", value);
 		}
 
+		[Fact]
+		public void CallbackWithIndexerGetter()
```

Comment:
I appreciate this test, and it would be good to have one like it (if we don't have one like it yet, I didn't double-check TBH)... but could you explain how it is related to this PR?

If it isn't directly related to this PR and you simply added it for completeness' sake, then I'd appreciate if you submitted it in a separate PR.